pkg/config/config.go

```diff
@@ -418,6 +444,98 @@ func NewConfig(userConfigPath string) (*Config, error) {
 	return config, nil
 }
 
+func hasLibpodConf() bool {
+	configs, err := systemLibpodConfigs()
+	return err == nil && len(configs) > 0
+}
+
+func (c *Config) libpodConfig() *ConfigFromLibpod {
+	var libpodConf ConfigFromLibpod
```

Comment:
How about you rewrite this as:

return libpodConf{
             SignaturePolicyPath: c.Containers.SignaturePolicyPath,
             InitPath: c.Containers.InitPath,
...
}